salt/client/ssh/ssh_py_shim.py

```diff
@@ -395,6 +404,11 @@ def main(argv):  # pylint: disable=W0613
         shutil.rmtree(OPTIONS.saltdir)
     else:
         retcode = subprocess.call(salt_argv)
+    if OPTIONS.wipe_state_file:
+        try:
+            os.remove(os.path.join(OPTIONS.saltdir, OPTIONS.wipe_state_file))
+        except OSError:
```

Comment:
Given our discussion above, I think this needs to be 
```suggestion
        except (IOError, OSError):
```
as well.

Reply:
Interesting, my lack of Python2 knowledge shows again. Thanks for the clarification!